dist/index.html

```diff
@@ -26,6 +19,15 @@ <h1>Add your score</h1>
 					<button id="add-btn" type="button"> Submit </button>			
 			</form>
         </div>
+        <div class="score_display" >
+            <div class="score_header">
+                <h1>Recent Scores</h1>
+                <button id="refresh_btn" type="button">Refresh</button>
+            </div>
+            <div class="score">
+            </div>
+        </div>
+    </div>
```

Comment:
- [ ] Please as it is a requirement for this project, sticking to the wireframe as much as possible, please kindly position the `add score section(form)` to the right, and score display(`Recent Scores`) to the left as in the wireframe(`see screenshot below`). Please also make sure the `headings`, `buttons` and `score display area` are aligned as in the wireframe. 

***The wireframe***
![LeaderWireframe](https://user-images.githubusercontent.com/65852621/157092879-7243ac7e-dfd9-41f3-bb9b-269bb9f4ff82.png)


***Your implementation
![LeaderImplementation](https://user-images.githubusercontent.com/65852621/157092907-27191c07-989d-43df-bfe3-52cbda02dd26.png)
***